src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.targets

```diff
@@ -18,7 +18,7 @@ Copyright (c) .NET Foundation. All rights reserved.
   </ImportGroup>
 
   <PropertyGroup>
-    <DefaultExcludes>bin\**;obj\**</DefaultExcludes>
+    <DefaultExcludes>bin\**;obj\**;packages\**</DefaultExcludes>
```

Comment:
could we add **\*.xproj as well? This would match exactly what we currently have in project.json world.